rules/awsrules/models/mappings/kinesis-firehose.hcl

```diff
@@ -10,8 +10,8 @@ mapping "aws_kinesis_firehose_delivery_stream" {
   redshift_configuration       = RedshiftDestinationConfiguration
   elasticsearch_configuration  = ElasticsearchDestinationConfiguration
   splunk_configuration         = SplunkDestinationConfiguration
-  cloudwatch_logging_options   = CloudWatchLoggingOptions
-  processing_configuration     = ProcessingConfiguration
-  processors                   = ProcessorList
-  parameters                   = ProcessorParameterList
+  // cloudwatch_logging_options   = CloudWatchLoggingOptions
+  // processing_configuration     = ProcessingConfiguration
+  // processors                   = ProcessorList
+  // parameters                   = ProcessorParameterList
```

Comment:
Some resources seem to provide schemas in another way. Here, I will comment out and update it later.